src/app/client/src/app/modules/uci-admin/uci-admin.module.ts

```diff
@@ -1,14 +1,14 @@
 import { NgModule} from '@angular/core';
 import { UciComponent } from './components/uci/uci.component';
 import { UciAdminRoutingModule } from './uci-admin-routing.module';
-import {UciModule} from '@samagra-x/uci-console';
+// import {UciModule} from '@samagra-x/uci-console';
 
 
 @NgModule({
   declarations: [UciComponent],
   imports: [
     UciAdminRoutingModule,
-    UciModule
+    // UciModule
```

Comment:
@princegupta1131 
removed commented code if not required

Reply:
Removed the commeted line